src/OpenLoco/src/Entities/EntityManager.h

```diff
@@ -38,9 +38,6 @@ namespace OpenLoco::EntityManager
 
     EntityId firstId(EntityListType list);
```

Comment:
L8-L11 can be removed. No need to forward declare vehicle head anymore.

Reply:
It's used for EntityTileList, can't remove that.